keyboards/tgr/tris/keymaps/default/keymap.c

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2020 halfenergized
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT_numpad_6x4(
+        KC_ESC,  KC_TAB,  MO(1),   KC_BSPC,
+        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,
+        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_P0,            KC_PDOT
+    ),
+
+    [1] = LAYOUT_numpad_6x4(
+        KC_TRNS,   KC_TRNS, KC_TRNS, MO(2),
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_HOME, KC_UP,   KC_PGUP,
+        KC_LEFT, KC_TRNS, KC_RGHT, KC_TRNS,
+        KC_END,  KC_DOWN, KC_PGDN, KC_TRNS,
+        KC_INS,           KC_DEL
+    ),
+
+    [2] = LAYOUT_numpad_6x4(
+        RESET,   KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,          KC_TRNS
+    )
```

Comment:
These are formatted wrong, and will produce the wrong layout when compiled. 

Specifically, this needs to be formatted to match: 
https://github.com/qmk/qmk_firmware/blob/master/layouts/default/numpad_6x4/default_numpad_6x4/keymap.c

```suggestion
    [0] = LAYOUT_numpad_6x4(
        KC_ESC,  KC_TAB,  MO(1),   KC_BSPC,
        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
        KC_P1,   KC_P2,   KC_P3,
        KC_P0,            KC_PDOT, KC_PENT
    ),

    [1] = LAYOUT_numpad_6x4(
        KC_TRNS,   KC_TRNS, KC_TRNS, MO(2),
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_HOME, KC_UP,   KC_PGUP,
        KC_LEFT, KC_TRNS, KC_RGHT, KC_TRNS,
        KC_END,  KC_DOWN, KC_PGDN,
        KC_INS,           KC_DEL,  KC_TRNS
    ),

    [2] = LAYOUT_numpad_6x4(
        RESET,   KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS,          KC_TRNS, KC_TRNS
    )
```